content/admin/administration-panel/Versions.adoc

```diff
@@ -0,0 +1,33 @@
+---
+Title: Version Control(Release)
+Weight: 80
+---
+
+:experimental:
+
+:imagesdir: ./../../../images/en/user
+
+= Version Control
+
+In this section you can manage the list of versions of your products. The list of versions is displayed in the drop-down list when describing a new
+link:../../../user/core-modules/bugs[bugs].
+
+To add a version to the list:
+
+ .	In the admin panel go to *Release*.
+ .	Click on the button *Create*.
+ .	In the sub-panel that appears, enter the following information:
+
+image:Release.png[Version control]
+
+Version:: Enter the required version number.
+Status:: From the drop-down list, select the status *Active* so that the version is displayed in the module
+link:../../../user/core-modules/bugs[bugs].
```

Comment:
BTW, I just noticed that on the paged linked from here, `user/core-modules/bugs/`, there are a few instances of the word "Appeals" where it should say "Cases"